modules/tensor_mechanics/src/materials/ComputeBeamEigenstrainFromAuxVar.C

```diff
@@ -0,0 +1,68 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ComputeBeamEigenstrainFromAuxVar.h"
```

Comment:
I think I'd re-name this "ComputeBeamEigenstrainFromVariable" because it could work with either an AuxVariable or a Variable. Also, change the parameter descriptions to reflect that fact as well (remove 'aux').